site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -1040,6 +1075,19 @@ components:
                description: Indicates if the northbound application was notified of the auto-provisioned device via an
                   empty event.
 
+      SearchTenantsResult:
+         type: object
+         description: The result of a search request for tenants.
+         additionalProperties: false
+         properties:
+            "total":
```

Comment:
can you please rebase and then add the *minimum* property analogously to the devices search result?

Reply:
I have added it.